administrator/components/com_banners/tables/banner.php

```diff
@@ -261,7 +261,7 @@ public function stick($pks = null, $state = 1, $userId = 0)
 		$k = $this->_tbl_key;
 
 		// Sanitize input.
-		JArrayHelper::toInteger($pks);
+		Joomla\Utilities\ArrayHelper::toInteger($pks);
```

Comment:
This isn't a 1-for-1 swap.  JArrayHelper took the variable by reference and the namespaced version doesn't.